core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.css

```diff
@@ -68,3 +68,10 @@
 .progress-nav-bar .row-item-previous-card {
   text-align: left;
 }
+
+.material-icons {
+  -moz-transform: scaleX(1) /*rtl:scaleX(-1)*/;
+  -o-transform: scaleX(1) /*rtl:scaleX(-1)*/;
+  -webkit-transform: scaleX(1) /*rtl:scaleX(-1)*/;
+  transform: scaleX(1) /*rtl:scaleX(-1)*/;
+}
```

Comment:
Can you explain this?

Reply:
This is for reversing the arrows, so the comment basically tells `rtlcss` to replace `scaleX(1)` with `scaleX(-1)` which would rotate the icons.

Reply:
Please add a comment explaining this and referencing the rtlcss docs.